packages/stokhos/src/sacado/kokkos/vector/Sacado_MP_Vector_ops.hpp

```diff
@@ -150,22 +150,26 @@ namespace Sacado {                                                      \
                                                                         \
       KOKKOS_INLINE_FUNCTION                                            \
       value_type val() const {                                          \
+        USING_OP                                                        \
         return OPER(expr.val());                                        \
```

Comment:
Use Kokkos mathematical functions, don't roll your own.
```suggestion
        return Kokkos::OPER(expr.val());                                        \
```